client/my-sites/post-selector/selector.jsx

```diff
@@ -459,12 +458,7 @@ class PostSelectorPosts extends React.Component {
 
 export default connect( ( state, ownProps ) => {
 	const { siteId, query } = ownProps;
-
-	const apiVersion =
-		! isJetpackSite( state, siteId ) || isJetpackMinimumVersion( state, siteId, '5.0' )
-			? '1.2'
-			: undefined;
-	const queryWithVersion = { ...query, apiVersion };
+	const queryWithVersion = { ...query, apiVersion: '1.2' };
```

Comment:
This is interesting. The `PostSelector` (used, for example, when selecting a post to copy in Classic Editor UI) uses v1.2 of the `/posts` API, while other parts of Calypso, like the list of pages or posts, uses v1.1.

The v1.2 endpoint was added by @marekhrabe in #11887 and D4766-code to hide private post types from the selector.

Today, the v1.2 endpoint has been around for almost 3 years and we could use it by default everywhere? That's probably a change in some Post Query Manager in `state/posts`?

Reply:
I'd be happy to do that, but it sounds like material for another PR 😉